app/app.js

```diff
@@ -151,16 +151,21 @@ angular.module('app',
             { map: {}, site: {}, features: {} };
     }])
     // inject the router instance into a `run` block by name
-    // .run(['$uiRouter', '$trace', '$location', function ($uiRouter, $trace, $location) {
+    //.run(['$uiRouter', '$trace', '$location', function ($uiRouter, $trace, $location) {
     //     // * uncomment this to enable the visualizer *
     //     let Visualizer = require('@uirouter/visualizer').Visualizer;
     //     let pluginInstance = $uiRouter.plugin(Visualizer);
     //     $trace.enable('TRANSITION');
     // }])
-    .run(['$rootScope', 'LoadingProgress', function ($rootScope, LoadingProgress) {
+    .run(['$rootScope', 'LoadingProgress', '$transitions', function ($rootScope, LoadingProgress, $transitions) {
         $rootScope.$on('$stateChangeError', console.log.bind(console));
```

Comment:
Does this even work?

Reply:
I don't think $stateChangeError or $stateChangeSuccess are recognized at all now. Should we remove this or replace with the $transitions equivalent?